docker-compose/.env.sample

```diff
@@ -0,0 +1,57 @@
+##############################
+### User Init Job Settings ###
+##############################
+# These variables will be used by the init jobs the first time you boot up with a fresh DB
+SUPER_ADMIN_EMAIL=
+SUPER_ADMIN_PASSWORD=
+INITIAL_REGISTRIES_JSON="eyJyZWdpc3RyaWVzIjpbeyJuYW1lIjoiRG9ja2VyaHViIiwiaG9zdG5hbWUiOiJkb2NrZXIuaW8iLCJhbGlhc2VzIjpbImluZGV4LmRvY2tlci5pbyJdLCJsb2dpbl9yZXF1aXJlZCI6ZmFsc2V9LHsibmFtZSI6IkdpdGh1YiBDb250YWluZXIgUmVnaXN0cnkiLCJob3N0bmFtZSI6ImdoY3IuaW8iLCJsb2dpbl9yZXF1aXJlZCI6ZmFsc2V9LHsibmFtZSI6Ikt1YmVybmV0ZXMgQ29udGFpbmVyIFJlZ2lzdHJ5IiwiaG9zdG5hbWUiOiJyZWdpc3RyeS5rOHMuaW8iLCJsb2dpbl9yZXF1aXJlZCI6ZmFsc2V9XX0="
+
+################
+### API KEYS ###
+################
+# The first time you boot up against a fresh DB, these will be used to configure the api keys for these service accounts
+# After that point, they will be used in the relevant pods for authentication with the API
+TRAWLER_API_KEY=111111
```

Comment:
For sample maybe we default to blank

Reply:
I can do that, and update the README to require ading those.

My logic for providing initial values is that since this sample environment is intended to be used for local development, the security wouldn't be a big concern. If they don't fill it in and boot up, then it will be randomly generated, and would probably be confusing to people.